e2e-tests/data-at-rest-encryption/conf/some-name.yml

```diff
@@ -9,7 +9,7 @@ spec:
 
   backup:
     enabled: true
-    image:
+    image: perconalab/percona-server-mongodb-operator:main-backup
```

Comment:
We need to use value from 'IMAGE_BACKUP' var. Does it work now?

Reply:
We use `IMAGE_BACKUP` **now**. See https://github.com/percona/percona-server-mongodb-operator/blob/main/e2e-tests/functions#L864, we need a string that ends with `-backup` here.